extension-manifest-v3/src/store/tab-stats.js

```diff
@@ -26,6 +30,16 @@ const Tracker = {
 const Stats = {
   domain: '',
   trackers: [Tracker],
+  trackersBlockedCount: ({ trackers }) =>
```

Comment:
It is better to return `Tracker` instances here, as then you can use them somehow, and still easily take a count from it - `stats.trackersBlocked.length`